src/foldermodel.cpp

```diff
@@ -122,7 +122,7 @@ void FolderModel::loadPendingThumbnails() {
     hasPendingThumbnailHandler_ = false;
     for(auto& item: thumbnailData_) {
         if(!item.pendingThumbnails_.empty()) {
-            auto job = new Fm::ThumbnailJob(item.pendingThumbnails_, item.size_);
+            auto job = new Fm::ThumbnailJob(std::move(item.pendingThumbnails_), item.size_);
```

Comment:
This is *again* about our different (or maybe, opposite) viewpoints, not about something real...